packages/middleware-sdk-glacier/src/add-glacier-api-version.spec.ts

```diff
@@ -9,10 +9,9 @@ describe("addGlacierApiVersion", () => {
   });
 
   it("sets the x-amz-glacier-version header", async () => {
-    const handler = addGlacierApiVersionMiddleware()(
-      mockNextHandler,
-      {} as any
-    );
+    const handler = addGlacierApiVersionMiddleware({
```

Comment:
If the function already lives in a `glacier` package, you don't need to mention `glacier` in your exported variable name again